### PR TITLE
Final logo regular in svg

### PR DESCRIPTION
**Pull request by phalem**

### CODE REVIEW

File: assets/logo/LLcheM.svg
Comment (MicPie) [inline assets/logo/LLcheM.svg:1]: @phalem The background in the reaction vessel is different in the three sag files. Maybe a copy and paste mistake?

**Comment by MicPie** [comment]
I closed the PR https://github.com/OpenBioML/chemnlp/pull/164 in favor of this one.

**Review by MicPie — CHANGES_REQUESTED** [review]

File: assets/logo/chemnlp.svg
[the remark this answers is not in the document]
Comment (phalem) [reply assets/logo/chemnlp.svg:1]: I make a combination of two logo which is Model and Dataset into chemnlp. So from the dataset I take the water science that model feed and from the model I take the cube which is our model structure. I hope this what you mean from this.

File: assets/logo/LLcheM.svg
Comment (phalem) [reply assets/logo/LLcheM.svg:1]: @MicPie This was a book inside the cube to make it different from chemnlp logo and llcheM logo(model logo)

File: assets/logo/chemnlp.svg
[the remark this answers is not in the document]
Comment (MicPie) [reply assets/logo/chemnlp.svg:1]: Yes, this is great, see my comment above.

**Comment by phalem** [comment]
> @phalem The logos look great!
> 
> Just two minor things I would like to change:
> 
> 1. Please remove the cap from the bottles. Then it looks a little bit cleaner. I already tried that and it looks a little bit better, see our slides where I already added it without cap: https://docs.google.com/presentation/d/1JkAKJveYsNGtAWoaksU8ykTdrC0aX3FshiFJ13SU6o8/edit#slide=id.g229d2eeb44b_2_0
> 2. Can you make the "ChemNLP" text only one line for the ChemNLP logo?
> 
> Thank you again for the great logo! I really like the book + liquid for the dataset and then the condensed block for the model. rocket

I will proceed in, sorry for being busy and far from the internet. I will PR it, I am sorry if this will take a while. I hope everyone like it.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md)
 
+<img src="/assets/logo/chemnlp.svg" width="200" height="200">
```
Comment (MicPie) [inline README.md:3]:
```suggestion
```

**Review by MicPie — APPROVED** [review]

**Pull request merged**